anuvaad-etl/anuvaad-extractor/file_translator/etl-file-translator/services/docx_transform.py

```diff
@@ -283,27 +292,33 @@ def generate_json_structure(self, document):
 
     def translate_paragraphs(self, para):
         try:
-            runs = common_obj.get_runs(para, para_obj=True)
-            para_id = common_obj.generate_id(file_id=self.file_id, para=str(self.sequence_para_index))
-            if para_id in self.trans_map:
-                self.distribute_over_runs(runs, trans_para=self.trans_map[para_id])
-            else:
-                raise FileErrors("translate_docx_file:", "PARA ID :{} not found in fetch content".format(para_id))
+            sub_para_counter = -1
+            for idx, runs in enumerate(common_obj.get_runs(para, para_obj=True)):
```

Comment:
get_runs returns a 2d list right, above you have named it subpara 
for idx, subpara in enumerate(common_obj.get_runs(para, para_obj=True)):
in this method you have named it runs. 
it may create confusion, so try to give same name if possible

Reply:
instead of using sub_para_counter use idx

Reply:
Will fix naming schemes to reflect similarly across all methods